.github/workflows/nightfalldlp.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
       - name: Add $GOPATH/bin
         run: |
-          echo ::add-path::$(go env GOPATH)/bin
+          echo $(go env GOPATH)/bin >> $GITHUB_PATH
```

Comment:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#adding-a-system-path

https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
![Screen Shot 2021-03-10 at 11 13 06 AM](https://user-images.githubusercontent.com/20913032/110660484-9d0c4480-8191-11eb-9e82-1c75915dedb3.png)